src/it/scala/ApiServiceISpec.scala

```diff
@@ -0,0 +1,146 @@
+    ()
+  }
+
+  override def afterAll(): Unit =
+    apiService.stop().unsafeToFuture()
+
+  override def afterEach(): Unit =
+    try
+      super.afterEach()
+    finally
+      if (Files.exists(Paths.get(replyMessage))) {
+        Files.delete(Paths.get(replyMessage))
+      }
+
+  private def readTextMessage(jmsMessage: JmsMessage): IO[Unit] =
+    jmsMessage.asTextF[IO].attempt.map {
+      case Left(e) => fail(s"Unable to read message file due to ${e.getMessage}")
+      case Right(text) =>
+        val file = Paths.get(replyMessage)
+        Using(Files.newBufferedWriter(file)) { writer =>
+          writer.write(text)
+        } match {
+          case Success(_) => ()
+          case Failure(e) => fail(s"Unable to write message file due to ${e.getMessage}")
+        }
+    }
+
+  def getFileContent: IO[String] =
+    IO.pure(getMessageFromFileStringPath(replyMessage))
+
+  def getMessageFromFileStringPath(messageFilePath: String): String =
+    Using(Source.fromFile(messageFilePath)) { source =>
+      source.getLines().mkString
+    } match {
+      case Success(value) => value
+      case Failure(_)     => ""
+    }
+
+  "The Message API" - {
+
+    "returns an echo message when a text message is sent with a SID of ECHO001" in {
+      val sqsTestConnector = SqsConnector("http://localhost:9324")
+      val sqsTestConnection: Connection = sqsTestConnector.getConnection
+      val session: Session = sqsTestConnection.createSession(false, Session.AUTO_ACKNOWLEDGE)
+      val searchQueue: String = "request-general"
+      val producer: MessageProducer = session.createProducer(session.createQueue(searchQueue))
+
+      val textMessage = session.createTextMessage("Hello World!")
+      textMessage.setStringProperty("sid", "ECHO001")
+      producer.send(textMessage)
+      eventually {
+        getFileContent.asserting(_ mustBe "The Echo Service says: Hello World!")
+      }
+    }
+
+    "returns an error message when the SID is not recognised" in {
+      pending // This test is marked pending until completion of https://national-archives.atlassian.net/browse/PACT-836
+      val sqsTestConnector = SqsConnector("http://localhost:9324")
+      val sqsTestConnection: Connection = sqsTestConnector.getConnection
+      val session: Session = sqsTestConnection.createSession(false, Session.AUTO_ACKNOWLEDGE)
+      val searchQueue: String = "request-general"
+      val producer: MessageProducer = session.createProducer(session.createQueue(searchQueue))
+
+      val textMessage = session.createTextMessage("Hello World!")
+      textMessage.setStringProperty("sid", "UNKNOWN")
+      producer.send(textMessage)
+      eventually {
+        getFileContent.asserting(_ mustBe "The Echo Service says: Hello World!")
+      }
+    }
+
+    "returns an error message when the message body is empty" in {
+      pending // This test is marked pending until completion of https://national-archives.atlassian.net/browse/PACT-836
```

Comment:
I'd just remove all of the contents below `pending` as it could mislead some.

Reply:
Removed.